src/generic/stage2/numberparsing.h

```diff
@@ -398,7 +349,9 @@ simdjson_really_inline error_code write_float(const uint8_t *const src, bool neg
   // digit count is off by 1 because of the decimal (assuming there was one).
```

Comment:
I'd remove this line since you explain it better down below.

Reply:
Agreed.

Reply:
Done.